lte/gateway/deploy/agw_install_docker.sh

```diff
@@ -48,9 +41,41 @@ ln -sf /var/run/systemd/resolve/resolv.conf /etc/resolv.conf
 sed -i 's/#DNS=/DNS=8.8.8.8 208.67.222.222/' /etc/systemd/resolved.conf
 service systemd-resolved restart
 
+echo 'debconf debconf/frontend select Noninteractive' | debconf-set-selections
+
+cat > /etc/apt/apt.conf.d/20auto-upgrades << EOF
+APT::Periodic::Update-Package-Lists "0";
+APT::Periodic::Download-Upgradeable-Packages "0";
+APT::Periodic::AutocleanInterval "0";
+APT::Periodic::Unattended-Upgrade "0";
+EOF
+
+apt purge --auto-remove unattended-upgrades -y
+apt-mark hold "$(uname -r)" linux-aws linux-headers-aws linux-image-aws
+
+
+# interface config
+INTERFACE_DIR="/etc/network/interfaces.d"
+mkdir -p "$INTERFACE_DIR"
+echo "source-directory $INTERFACE_DIR" > /etc/network/interfaces
+
+# get rid of netplan
+systemctl unmask networking
+systemctl enable networking
+
 echo "Install Magma"
 apt-get update -y
-apt-get install curl zip python3-pip docker.io -y
+apt-get upgrade -y
+apt-get install curl zip python3-pip docker.io net-tools -y
+
+echo "Making sure $MAGMA_USER user is sudoers"
+if ! grep -q "$MAGMA_USER ALL=(ALL) NOPASSWD:ALL" /etc/sudoers; then
+  apt install -y sudo
```

Comment:
We could install sudo with the other packages in line 69 to avoid this separate apt usage.